### PR TITLE
AC New Test - LPD-29404 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
Jira ticket: https://liferay.atlassian.net/browse/LPD-29404

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by marcosapmf** [comment]
ci:report:33625902

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:sf

File: modules/test/playwright/tests/osb-faro-web/utils/utils.ts [hunk 1]
```diff
@@ -35,12 +27,30 @@ export async function expectToBeVisible({
 	itemNames,
 	page,
 }: {
-	itemNames: string[];
+	itemNames: string[] | string;
 	page: Page;
 }) {
-	for (const itemName of itemNames) {
-		await expect(page.getByRole('cell', {name: itemName})).toBeVisible({
+	const namesArray = Array.isArray(itemNames) ? itemNames : [itemNames];
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/utils/utils.ts:33]: rename to `names`

File: modules/test/playwright/tests/osb-faro-web/utils/utils.ts [hunk 2]
```diff
@@ -5,27 +5,19 @@
...
 	itemNames,
 	page,
 }: {
-	itemNames: string[];
+	itemNames: string[] | string;
 	page: Page;
 }) {
-	for (const itemName of itemNames) {
-		await expect(page.getByRole('cell', {name: itemName})).toBeHidden({
+	const namesArray = Array.isArray(itemNames) ? itemNames : [itemNames];
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/utils/utils.ts:17]: rename to `itemNamesArray`

Comment (ViniLopes20) [reply modules/test/playwright/tests/osb-faro-web/utils/utils.ts:17]: Done

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**